integration_test/app_test.dart

```diff
@@ -79,14 +79,82 @@ void main() async {
     String surveyTitleText = 'ibis Bangkok Riverside';
     expect(find.text(surveyTitleText), findsOneWidget);
 
-    final surveyDetailsButtonFinder =
-        find.byKey(const Key(WidgetKeys.surveyDetailsButtonKey));
-    expect(surveyDetailsButtonFinder, findsOneWidget);
-    await tester.tap(surveyDetailsButtonFinder);
+    final nextButtonFinder = find.byKey(const Key(WidgetKeys.nextButtonKey));
+    expect(nextButtonFinder, findsOneWidget);
+    await tester.tap(nextButtonFinder);
     await tester.pumpAndSettle();
 
     final startSurveyButtonFinder =
         find.byKey(const Key(WidgetKeys.startSurveyButtonKey));
     expect(startSurveyButtonFinder, findsOneWidget);
   });
+
+  testWidgets('Form screen test', (WidgetTester tester) async {
+    await tester.pumpWidget(const ProviderScope(
+      child: SurveyApp(),
+    ));
+    await tester.pumpAndSettle();
+
+    final nextButtonFinder = find.byKey(const Key(WidgetKeys.nextButtonKey));
+    await tester.tap(nextButtonFinder);
+    await tester.pumpAndSettle();
+
+    final startSurveyButtonFinder =
+        find.byKey(const Key(WidgetKeys.startSurveyButtonKey));
+    await tester.tap(startSurveyButtonFinder);
+    await tester.pumpAndSettle();
+
+    // star
+    await tester.tap(nextButtonFinder);
+    await tester.pumpAndSettle();
```

Comment:
Should we expect to choose another option instead of using default option when doing the UI test? Apply for other above questions too 🙏

Reply:
From my perspective, we should not allow default options for the user to blindly navigate to the next questions like this 🙏